docs/release-promotion/desktop/howto-rc.md

```diff
@@ -1,29 +1,37 @@
 # Add a What's New Page to a Firefox RC
 
+## When
+
+What's New Page setup should be done shortly after updates are available on the release-localtest channel.
+
 ## Requirements
 
 * Access to production Balrog
 * A confirmed list of locales that should receive the WNP from Product
 * A confirmed URL of the WNP from Product
 
-## How
+## wnp
+### How
 
 * Download the existing Release Blob (e.g. `Firefox-59.0-build1`)
   * Go to https://aus4-admin.mozilla.org/releases
   * Search for the release blob you need
   * Click `Download`
-* Make the following changes to it:
+  * Copy it to a -No-WNP version (you'll need this later). Eg: Firefox-59.0-build1-No-WNP.json.
```

Comment:
Let's include in `-No-WNP` and `Firefox-59.0-build1-No-WNP.json` for better highlighting.

Reply:
Are you saying we should check in sample blobs?

Reply:
No, sorry, I just meant syntax highlighting here. As in `-No-WNP` instead of simply -No-WNP. I find it easier to copy-paste stuff when it's highlighted like that.